Resources/Prototypes/_SpaceCats/Entities/Clothing/Uniforms/jumpsuits.yml

```
@@ -0,0 +1,32 @@
+    - type: Sprite
+      sprite: _SpaceCats/Clothing/Uniforms/Jumpsuit/captain_white.rsi
+    - type: Clothing
+      sprite: _SpaceCats/Clothing/Uniforms/Jumpsuit/captain_white.rsi
+      
+- type: entity
+  parent: ClothingUniformBase
+  id: ClothingUniformJumpsuitSheriff
+  name: sheriff jumpsuit
+  description: Jumpsuit showing your status is how terrible... the great ruler.
+  components:
+    - type: Sprite
```

yamllint (1.29.0-1)

[warning] wrong indentation: expected 2 but found 4

(indentation)